### PR TITLE
Helpers

### PR DESCRIPTION
**Pull request by shivam-tripathi**
Add relevant helper functions.

### CODE REVIEW

File: src/helpers/config.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+/*
+ * @type {string} FILE
+ * Path to config file.
+ **/
+const FILE = path.join(__dirname, '../../config/config.json');
+
+/**
+ * Extract the import configuration from the config file.
+ * @param {Array.<string>} configKeys - The keys, in nesting order, to reach the
```
Comment (LordSputnik) [inline src/helpers/config.js]: Extra dot here - should be `Array<string>`

File: src/helpers/config.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ *		relevant config object. For example, to extract OL works, if the JSON
+ *		object structure is {"openLibrary": {"works": {...}, ...}, ...}, the
+ *		arguments would be ['openLibrary', 'works'].
+ * @returns {Object} - JS Object constaining configuration values.
+ */
+export default function config(configKeys) {
+	try {
+		if (configKeys || null) {
```
Comment (LordSputnik) [inline src/helpers/config.js]: `if(configKeys || null)` is equivalent to just `if(configKeys)` - if it evaluates as falsey for the boolean expression, it'll evaluate as falsey for the if statement too.

File: src/helpers/utils.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+import _ from 'lodash';
+
+
+/**
+ * mergeSets: Merge an array of sets into a single set.
+ * @param {Array.<Set>} setArr - Array containing a number of sets.
+ * @returns {Set} - Set containing all unique values from the array of sets.
+ */
+export function mergeSets(setArr) {
```
Comment (LordSputnik) [inline src/helpers/utils.js]: Smart, I like this :+1:

File: src/helpers/utils.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+	}());
+}
+
+/**
+ * isNotDefined: Check if a variable is undefined, null or contains
+ * 		any other falsy value
+ * @param {?} value - The variable to be checked for definition
+ * @returns {boolean} - Boolean value if the variable is not defined or not.
+ */
+export function isNotDefined(value) {
```
Comment (LordSputnik) [inline src/helpers/utils.js]: This shouldn't be needed - you should be able to do `Boolean(value)` to achieve the same (or just evaluate the value directly if it's in an if statement):

    > Boolean(undefined)
    false
    > Boolean(null)
    false
    > Boolean(false)
    false
    > Boolean(0)
    false

Comment (shivam-tripathi) [reply src/helpers/utils.js]: I'll open an issue for this, and make changes for it in the end.

**Pull request merged**